Angular/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  root: true,
+  extends: [
+      'devextreme/angular'
+  ]
+};
```

Comment:
Let's move all [project specific parameters](https://github.com/DevExpress/eslint-config-devextreme/pull/41/files#diff-f87bc0f2a3e5d6636d97efc50dc7797a06ef33f48e328df03efdd86d6e05bd90R40-R45) from eslint config here

Something like this:
```suggestion
module.exports = {
  root: true,
  extends: ['devextreme/spell-check'],
  overrides: [{
    files: ['*.ts', '*.html'],
    extends: ['devextreme/angular'],
    env: { es6: true },
    parserOptions: {
      project: './tsconfig.json',
      'createDefaultProgram': true,
      'ecmaVersion': 6,
    },
  }]
};
```